### PR TITLE
fix!: adds blueprint metadata and TPG >= 3.53

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -20,7 +20,17 @@ terraform {
...
+    }
+
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 3.53, < 5.0"
+    }
+
+    random = {
+      source  = "hashicorp/random"
+      version = "~>3.4"
```
Comment (apeabody) [inline versions.tf:33]: To reduce the likelihood of impact, I'd suggest:

```suggestion
      version = ">= 2.1"
```

File: versions.tf [hunk 2]
```diff
@@ -20,7 +20,17 @@ terraform {
     google = {
       source = "hashicorp/google"
       # TODO: Check why this filter was breaking tests.
-      # version = "~> 3.53, < 5.0"
+      version = ">= 3.53, < 5.0"
```
Comment (apeabody) [inline versions.tf:23]: Looks like this TPG version constraint is technically new, so we should probably mark this as a breaking change.

**Review by apeabody — APPROVED** [review]
Note the recommendation on the random provider, otherwise LGTM as a breaking fix.

**Comment by g-awmalik** [comment]
Stale version of metadata.

**Pull request closed without merging**